lte/gateway/configs/BUILD.bazel

```diff
@@ -0,0 +1,72 @@
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@rules_pkg//pkg:mappings.bzl", "pkg_filegroup", "pkg_files", "pkg_mklink")
+
+pkg_filegroup(
+    name = "magma_config_files",
+    srcs = [
+        ":lte_configs",
+        ":pipelined_symlink",
+        ":sessiond_symlink",
+    ],
+    visibility = ["//lte/gateway/release:__pkg__"],
+)
+
+pkg_mklink(
+    name = "pipelined_symlink",
+    link_name = "/etc/magma/pipelined.yml",
+    target = "/etc/magma/pipelined.yml_prod",
```

Comment:
Could we just store `pipelined.yl_prod` at `/etc/magma/pipelined.yml` instead of symlinking?

Reply:
Would be possible, yes - for now this is the behavior of the make created .deb.
I can only speculate: I assume the reason is/was, to have a convenient way to switch between prod and test setups.